-------------------------------------------------------------------------------
Test set: com.parkit.parkingsystem.dao.TicketDAOTest
-------------------------------------------------------------------------------
Tests run: 7, Failures: 1, Errors: 2, Skipped: 0, Time elapsed: 0.024 s <<< FAILURE! - in com.parkit.parkingsystem.dao.TicketDAOTest
testSimpleUpdateTicket  Time elapsed: 0.001 s  <<< ERROR!
org.mockito.exceptions.base.MockitoException: 

Only void methods can doNothing()!
Example of correct use of doNothing():
    doNothing().
    doThrow(new RuntimeException())
    .when(mock).someVoidMethod();
Above means:
someVoidMethod() does nothing the 1st time but throws an exception the 2nd time is called
	at com.parkit.parkingsystem.dao.TicketDAOTest.testSimpleUpdateTicket(TicketDAOTest.java:157)

testSaveTicket  Time elapsed: 0 s  <<< FAILURE!
org.opentest4j.AssertionFailedError: expected: <true> but was: <false>
	at com.parkit.parkingsystem.dao.TicketDAOTest.testSaveTicket(TicketDAOTest.java:62)

testSaveTicketException  Time elapsed: 0.01 s  <<< ERROR!
java.lang.VerifyError: 
Bad return type
Exception Details:
  Location:
    org/mockito/internal/junit/ExceptionFactory$JUnitArgsAreDifferent.create(Ljava/lang/String;Ljava/lang/String;Ljava/lang/String;)Ljava/lang/AssertionError; @18: areturn
  Reason:
    Type 'org/mockito/exceptions/verification/junit/ArgumentsAreDifferent' (current frame, stack[0]) is not assignable to 'java/lang/AssertionError' (from method signature)
  Current Frame:
    bci: @18
    flags: { }
    locals: { 'java/lang/String', 'java/lang/String', 'java/lang/String', '[Z' }
    stack: { 'org/mockito/exceptions/verification/junit/ArgumentsAreDifferent' }
  Bytecode:
    0000000: b800 224e bb00 0259 2a2b 2cb7 0003 2d04
    0000010: 0454 b0                                

	at com.parkit.parkingsystem.dao.TicketDAOTest.testSaveTicketException(TicketDAOTest.java:83)

